testng-core/src/main/java/org/testng/internal/IConfiguration.java

```diff
@@ -49,4 +49,10 @@ default boolean addExecutionListenerIfAbsent(IExecutionListener l) {
   boolean getOverrideIncludedMethods();
 
   void setOverrideIncludedMethods(boolean overrideIncludedMethods);
+
+  default void setReportAllDataDrivenTestsAsSkipped(boolean reportAllDataDrivenTestsAsSkipped) {}
+
+  default boolean getReportAllDataDrivenTestsAsSkipped() {
+    return false;
```

Comment:
4th place with the default value

Reply:
This is an interface with a default implementation. There's no way to circumvent this part without making the method as `non default`. So I guess we should be fine with this one.

Reply:
You can use a constant for the default value.

Or you can use a classical interface.